### ISSUE TITLE
Quality issues with Inpainting 

### ISSUE THREAD

**Issue by imperator-maximus** [Stability-AI/stability-sdk#146]
Hello,

the current Inpainting implementation changes the whole colour of its surrounding at white tones.
I tried it on Dreamstudio and API and can be easily reproduced.
I tried it with Inpainting 1.0 and 2.0 
![image](https://user-images.githubusercontent.com/4406586/204152557-1dd81c23-7c9f-435d-abe1-119da6a5455c.png)

**Comment by imperator-maximus** [comment]
Another issue is still the content filter which is known already to have many false positives. Getting this in Inpainting quite often while generating hands for example:
![image](https://user-images.githubusercontent.com/4406586/206935100-20f3815d-30fc-462d-b203-c6d143ea3b3a.png)

**Comment by johnsabath** [comment]
To confirm, you were seeing this through the SDK/gRPC API?

Do you have an example of a mask image that produced these artifacts? (I'm mostly curious if there was a hard line or smooth gradient for the mask edges).

**Comment by imperator-maximus** [comment]
yes it is gRPC I just wanted to send you a mask image but now I doesn't work at all so I made a new ticket: https://github.com/Stability-AI/stability-sdk/issues/174

**Comment by dubtor** [comment]
I can confirm this behavior. Using the Python SDK. I hope it will be fixed
